src/seedu/addressbook/AddressBook.java

```diff
@@ -1151,7 +1155,19 @@ private static String getUsageInfoForExitCommand() {
      * @return  string without the sign
      */
     private static String removePrefixSign(String s, String sign) {
```

Comment:
as discussed in class, the term`sign` is not common (or in fact used elsewhere in this program) and we may want to just stick to the term `prefix` instead.